README.md

```diff
@@ -366,6 +382,14 @@ export interface NodeOptions {
 
   // A mapping of service names to their respective handlers.
   services?: { [key: string]: Handler };
+
+  // Total number of attempts to reconnect to a peer we reached that disconnected.
+  // Default is 8 attempts, set to 0 to not attempt to reconnect at all.
+  numReconnectAttempts?: number;
```

Comment:
Can 0 be no reconnect and -1 be forever retry?

Reply:
Yep! Updated it for the NodeJS SDK.